__tests__/app.test.js

```diff
@@ -169,13 +192,10 @@ describe.skip("GET /api/articles/:article_id", () => {
         created_at: '2020-07-09T20:11:00.000Z',
         votes: 100,
         article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
-        
-          
+        comment_count: 11         
```

Comment:
Usually, we don't really want to go back and change our tests. A new one would've been good!